### PR TITLE
Fix manual send SystemRequest(QUERY_APP) from mobile app

### PR DESCRIPTION
**Pull request by AGritsevich**
It was implemented not approved req-t that change current rules
to validate Json file. Excess code was deleted and created link in
req-ts.
Please, for more details see comments in attached task.
Closes-bug: [APPLINK-24365](https://adc.luxoft.com/jira/browse/APPLINK-24365).

Please review @AByzhynar , @AGaliuzov , @OHerasym , @nk0leg , @LuxoftAKutsan , @anosach-luxoft

### CODE REVIEW

File: src/components/application_manager/src/commands/mobile/system_request.cc [hunk 1]
```diff
@@ -85,7 +85,7 @@ class QueryAppsDataValidator {
     if (!HasResponseKey()) {
       return false;
     }
-    return ValidateAppDataAndOsAndLanguagesData();
+    return true;
```
Comment (AByzhynar) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @AGritsevich Why did you replace big function with lots of logic with one const bool value ?

Comment (AGritsevich) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @AByzhynar , do you read pull request message?

Comment (AByzhynar) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @AGritsevich I've read it. I'm just wondering when the reqs will be approved - are you going to return back this portion of code? Where are you going to keep it?

Comment (AGritsevich) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @AByzhynar , in Jira with possible task to implement it.

**Comment by LuxoftAKutsan** [comment]
Reviewed

**Comment by OHerasym** [comment]
Reviewed

**Comment by nk0leg** [comment]
Reviewed

**Comment by AByzhynar** [comment]
Reviewed

**Comment by AGritsevich** [comment]
Please review @AGaliuzov, @anosach-luxoft

File: src/components/application_manager/src/commands/mobile/system_request.cc [hunk 2]
```diff
@@ -105,96 +106,6 @@ class QueryAppsDataValidator {
     return true;
   }
 
-  bool ValidateAppDataAndOsAndLanguagesData() const {
```
Comment (anosach-luxoft) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @AGritsevich do we use other methods from this class anywhere?

Comment (AGritsevich) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @anosach-luxoft , good catch. `ValidateLanguages`, `ValidateAppIdAndAppName` never used now. I'll delete it too.

Comment (AGritsevich) [inline src/components/application_manager/src/commands/mobile/system_request.cc]: @anosach-luxoft , see 85fa792

**Comment by anosach-luxoft** [comment]
Reviewed

**Comment by AGaliuzov** [comment]
Reviewed

**Pull request merged**